mainnet-artifacts.bzl

```diff
@@ -0,0 +1,59 @@
+"""
+This module defines Bazel targets for the mainnet versions of some of the published binaries (/publish/binaries)
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+# This variable is automatically kept in sync by ci/scripts/update-mainnet-artifacts.sh
+# with the IC version (git revision) of the mainnet NNS subnet (tdb26-...) tracked in testnet/mainnet_revisions.json
+MAINNET_NNS_SUBNET_IC_VERSION = "99ab7f03700ba6cf832eb18ffd55228f56ae927a"
+
+# Which published binaries to download. This dictionary maps the name of the binary to a dictionary of type:
+# {"rev":"<rev>", "sha256":"<sha256>"} where
+#   <rev>: is the git revision of the published binary.
+#     This should always be set to MAINNET_NNS_SUBNET_IC_VERSION
+#     unless there's a strong reason to override it. For example when the binary is incompatible with HEAD.
+#   <sha256>: this should be the SHA256 hash of the published gz-compressed binary.
+#     If <rev> equals MAINNET_NNS_SUBNET_IC_VERSION this hash is automatically updated
+#     by ci/scripts/update-mainnet-artifacts.sh.
+PUBLISHED_BINARIES = {
```

Comment:
```suggestion
# Please make sure the key value pairs are contained on a single line otherwise ci/scripts/update-mainnet-artifacts.sh will fail to update them correctly.
PUBLISHED_BINARIES = {
```